lldb/source/Target/Process.cpp

```diff
@@ -6528,6 +6528,74 @@ static void AddRegion(const MemoryRegionInfo &region, bool try_dirty_pages,
                 CreateCoreFileMemoryRange(region));
 }
 
+static void AddRegisterSections(Process &process, ThreadSP &thread_sp,
```

Comment:
I assume this is adding "Thread Local Data Memory Region" which can be read later by `DynamicLoader::GetThreadLocalData()` API? If so , we should make the function name meaningful. Let's call it `AddThreadLocalDataRegion`. There is no memory region called "register sections".